malachite-nz/Cargo.toml

```diff
@@ -23,7 +23,7 @@ path = "src/bin.rs"
 [dependencies]
 embed-doc-image = "0.1.4"
 itertools = "0.9.0"
-malachite-base = "0.3.2"
```

Comment:
Is there any reason local dependencies refers the public version instead of local version?